tutorials/08-tdem/plot_fwd_1_em1dtm_stitched_sk.py

```diff
@@ -0,0 +1,234 @@
+"""
```

Comment:
@jcapriot: rewrote the example. have a look and let me know if this makes sense to you.